apps/volontulo/views/organizations.py

```diff
@@ -65,6 +67,19 @@ def post(request):
                 {'organization': Organization()}
             )
 
+        try:
+            validate_email(request.POST.get('address'))
```

Comment:
ok I know that it is not your code but I really don't like it, its totally wrong and should be validated by django forms. This whole class should be rewrited.